### DISCUSSION TITLE
Does it work well in the mobile app?

### DISCUSSION THREAD

**Discussion by jroachgit** [st3v3nmw/obsidian-spaced-repetition#202] [Q&A]
Hi All,

How does the deck work in the mobile app?

I've mainly used anki integration -- until I've seen this plugin.

**Comment by st3v3nmw** [comment]
For me it does, save for one flashcard modal size issue (#231) that can easily be fixed in settings.

**Comment by j-r-e-i-d** [comment]
I can’t get it to work at all on mobile. 
Tapping the icon in the ribbon on the left does nothing.

**Reply by AB1908** [reply]
Can you try perhaps enabling a mobile console and then checking the logs? You can use BRAT to install the devtools console thing. You should be able to find some more instructions on the Discord.
